Components/Navigation/navigation.html

```diff
@@ -88,10 +88,10 @@ <h1>App</h1>
           </ul>
         </nav>
 
-        <br />
-        <br />
-        <br />
         <p>
+          <br />
+          <br />
+          <br />
           Lorem Ipsum is simply dummy text of the printing and typesetting

```

Comment:
Use padding insted of `br` tag as it is not semantic as well as bad for screen readers. [Link](https://stackoverflow.com/questions/1726073/is-it-sometimes-bad-to-use-br#:~:text=The%20main%20reason%20for%20not,space%20the%20blocks%20out%20properly.)

Reply:
Agreed, will fix

Reply:
@thegeorgenikhil  Removed all br tags from the code